app/views/support_interface/references/cancel.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :browser_title, 'Cancel reference' %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(@reference.application_form)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Are you sure you want to cancel this reference?
+    </h1>
+
+    <p class="govuk-body">
+      Cancelling a reference will:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>prevent this referee from providing a reference</li>
+      <li>allow the candidate to provide a replacement referee</li>
+    </ul>
+
+    <p class="govuk-body">
+      Cancelling a reference does <strong>not send email notifications</strong>.
```

Comment:
Non-blocking design crit: [design system says to use bold sparingly](https://govuk-design-system-preview.netlify.com/styles/typography/#bold-text), not sure if it is worth it here. cc @fofr